bigbluebutton-html5/imports/ui/components/nav-bar/component.jsx

```diff
@@ -44,12 +46,20 @@ const intlMessages = defineMessages({
     id: 'app.navBar.recording.off',
     description: 'label for indicator when the session is not being recorded',
   },
+  startTitle: {
+    id: 'app.recording.startTitle',
+    description: 'start recording title',
+  },
+  stopTitle: {
+    id: 'app.recording.stopTitle',
+    description: 'stop recording title',
+  },
 });
 
 const propTypes = {
   presentationTitle: PropTypes.string,
   hasUnreadMessages: PropTypes.bool,
-  recordProps: PropTypes.objectOf(PropTypes.bool),
+  recordProps: PropTypes.objectOf(PropTypes.any),
```

Comment:
I think something like this would be better than just checking for any propType
```
recordProps: PropTypes.shape({
    time: PropTypes.number,
    recording: PropTypes.bool,
}),
```